src/test/java/org/candlepin/subscriptions/capacity/admin/InternalSubscriptionResourceTest.java

```diff
@@ -222,6 +224,21 @@ void shouldReturnActiveSubscriptionAndNotTerminated_WhenOrgIdPresent() {
     assertEquals("bar3", awsUsageContext.getAwsSellerAccountId());
   }
 
+  @Test
+  void azureUsageContextEncodesAttributes() {
+    var endDate = OffsetDateTime.of(2022, 1, 1, 6, 0, 0, 0, ZoneOffset.UTC);
+    Subscription sub = new Subscription();
+    sub.setBillingProviderId("resourceId;planId;offerId");
+    sub.setEndDate(endDate);
+    when(syncController.findSubscriptions(any(), any(), any(), any())).thenReturn(List.of(sub));
+    var azureUsageContext =
+        resource.getAzureMarketplaceContext(
+            "org123", endDate, "rhosak", "Premium", "Production", "123");
```

Comment:
can we make this something other than "rhosak"?  we're working on rhosak removal